x-pack/plugins/security_solution/public/cases/components/case_view/index.tsx

```diff
@@ -387,30 +388,37 @@ export const CaseComponent = React.memo<CaseProps>(
                       caseUserActions={caseUserActions}
                       connectors={connectors}
                       data={caseData}
-                      fetchUserActions={fetchCaseUserActions.bind(null, caseData.id)}
+                      fetchUserActions={fetchCaseUserActions.bind(null, caseId, subCaseId)}
                       isLoadingDescription={isLoading && updateKey === 'description'}
                       isLoadingUserActions={isLoadingUserActions}
                       onShowAlertDetails={showAlert}
                       onUpdateField={onUpdateField}
                       updateCase={updateCase}
                       userCanCrud={userCanCrud}
                     />
-                    <MyEuiHorizontalRule margin="s" />
-                    <EuiFlexGroup alignItems="center" gutterSize="s" justifyContent="flexEnd">
-                      <EuiFlexItem grow={false}>
-                        <StatusActionButton
-                          status={caseData.status}
-                          onStatusChanged={changeStatus}
-                          disabled={!userCanCrud}
-                          isLoading={isLoading && updateKey === 'status'}
+                    {(caseData.type !== CaseType.collection || hasDataToPush) && (
```

Comment:
I was trying to find a better way to do this instead of having the double checks but couldn't think of one haha 👍

Reply:
Me too. I fought  a lot with this :)